mlos_bench/mlos_bench/config/environments/apps/fio/scripts/remote/latency-profile.fio

```diff
@@ -0,0 +1,21 @@
+# Test job that demonstrates how to use the latency target
+# profiling. Fio will find the queue depth between 1..128
+# that fits within the latency constraints of this 4k random
+# read workload.
+
+[global]
+bs=4k
+rw=randrw
+random_generator=lfsr
+direct=1
+ioengine=libaio
+iodepth=128
+# Set max acceptable latency to 500msec
+latency_target=500000
+# profile over a 20s window
+latency_window=20000000
+# 99.0% of IOs must be below the target
+latency_percentile=99.0
+
+[device]
+filename=/dev/sda
```

Comment:
This should probably be controllable.  One option would be to let it run in the CWD, and adjust that in the `run` script above.

Reply:
yes, I'll move it to the CLI level and expose the parameter in `const_args`